server/status/monitor.go

```diff
@@ -94,6 +98,13 @@ func (nsm *NodeStatusMonitor) VisitStoreMonitors(visitor func(*StoreStatusMonito
 func (nsm *NodeStatusMonitor) StartMonitorFeed(feed *util.Feed) {
 	sub := feed.Subscribe()
 	go storage.ProcessStoreEvents(nsm, sub)
+	sub = feed.Subscribe()
```

Comment:
I'm not even sure I need the variable - I just wanted the Subscribe() statement to evaluate before the goroutine starts, but I think that happens if I call it inline:

`go storage.ProcessStoreEvents(nsm, feed.Subscribe())`